configure.ac

```diff
@@ -35,6 +35,7 @@ AX_CHECK_COMPILER_FLAGS(-pipe, [OBJCFLAGS="$OBJCFLAGS -pipe"])
 AX_CHECK_COMPILER_FLAGS(-fno-common, [OBJCFLAGS="$OBJCFLAGS -fno-common"])
 AX_CHECK_COMPILER_FLAGS(-Wobjc-missing-property-synthesis -Werror,
 	[OBJCFLAGS="$OBJCFLAGS -Wobjc-missing-property-synthesis"])
+AX_CHECK_COMPILER_FLAGS(-Wno-ignored-attributes, [OBJCXXFLAGS="$OBJCXXFLAGS -Wno-ignored-attributes"])
```

Comment:
You need to check it together with -Werror, because otherwise the compiler could issue a warning that it doesn't know that warning and the check succeeds. However, when adding it, do not add the -Werror, as it already is added later on.

Also note that you go over 80 columns here and could break it like in the line before.